pylua/timer.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# timer works with milliseconds
+
+import threading
+
+
+def convert_milliseconds_to_seconds(milliseconds):
+    milliseconds_in_second = 1000.0
+    return milliseconds/milliseconds_in_second
+
+
+def convert_seconds_to_milliseconds(seconds):
+    milliseconds_in_second = 1000.0
+    return seconds*milliseconds_in_second
+
+
+class PyTimer(object):
+
+    def __init__(self, task, interval):
+        self._task = task
+        self._interval = convert_milliseconds_to_seconds(interval)
+        self._timer = None
+
+    def start(self, interval=None):
+        if interval is not None:
```

Comment:
@dtrunov, why do you need two methods (ctor, start) which allow to set interval? On my opinion this is redundant. Keep only ctor. If someone will need to change interval then timer can be recreated. This way test will be simpler. I see no point in logic duplication..